### DISCUSSION TITLE
Decoupling the Form component from Sveltekit to enhance reusability

### DISCUSSION THREAD

**Discussion by ruizdiazever** [huntabyte/shadcn-svelte#1227] [Ideas]
After the resounding success of Astro and its agnotism, many of us have decided to use Svelte with Astro, for many reasons... and shadcn-svelte is one of the most important in the Svelte ecosystem, it would be great if all shacdn-svelte were agnostic in this sense too! Is there a plan or roadmap for this?

**Comment by maxiCalderonBuono** [comment]
Have been looking for this for a while. It would be amazing to have some adapter or alternative to how it works right now. Looking forward to hearing some news.

**Comment by Vale92882** [comment]
Are there any ongoing plans ? It would be great to see shadcn-svelte embrace this direction.

**Comment by huntabyte** [comment]
This is not currently in the plans as the underlying library that powers the core form functionality depends heavily on SvelteKit.
